bip-0003.md

```diff
@@ -710,12 +712,12 @@ feedback, and helpful comments.
     header with the "Proposed-Replacement" header that lists any proposals that recommend replacing the original document.
 [^proposes-to-replace]: **Why was "Replaces" retained instead of changing it to "Proposes-to-Replace"?**  
     When one BIP proposes to supersede another, it is on the original BIP where things get complicated. The BIP is an
-    author document, but depending on its progress through the Workflow, it may be meanwhile co-owned by community. Who gets to decide
+    author document, but depending on its progress through the Workflow, it may be meanwhile co-owned by community. Who may decide
```

Comment:
```suggestion
    author document, but depending on its progress through the workflow, it may meanwhile be co-owned by the community. Who may decide
```